js/services/engine.ts

```diff
@@ -980,7 +980,7 @@ export const move = (outcome: any, flowKey: string)  => {
 
     const invokeRequest = Json.generateInvokeRequest(
         State.getState(flowKey),
-        'FORWARD',
+        Model.getInvokeType(flowKey),
```

Comment:
Comment on line 953 needs to be changed to better reflect what is happening.

Reply:
Fixed thanks